src/SwedbankPay.Sdk.Infrastructure/TransactionDto.cs

```diff
@@ -34,6 +34,12 @@ internal ITransaction Map()
             var state = string.IsNullOrEmpty(State) ? "Unknown" : State;
             var id = new Uri(Id, UriKind.RelativeOrAbsolute);
 
+            IProblem problem = null;
+            if(Problem != null)
+            {
+                problem = Problem.Map();
+            }

```

Comment:
Can't you just use `null` propagation?

```suggestion
            var problem = Problem?.Map();
```

Reply:
I can!
I love `null` propagation but I keep forgetting it exists.